src/javascript/app/base/header.js

```diff
@@ -92,6 +108,9 @@ const Header = (() => {
         applyToAllElements('.url-manage-account', el => {
             el.href = Url.urlForDeriv('redirect', `action=manage_account&ext_platform_url=${encodeURIComponent(window.location.href)}`);
         });
+        applyToAllElements('.url-wallets-deposit', el => {
+            el.href = Url.urlForDeriv('wallets/cashier/transfer', `ext_platform_url=${encodeURIComponent(window.location.href)}`);
```

Comment:
Will it make sense to move `encodeURIComponent(window.location.href)` in a constant and reuse it? `encodeURIComponent(window.location.href)` is used in multiple lines in `setHeaderUrls`

Reply:
ya this is fine. i'll implement.